src/components/slop-fest-modal.jsx

```diff
@@ -0,0 +1,210 @@
+import { useMutation, useQuery } from "@apollo/client";
+import { useContext, useEffect, useState } from "react";
+import { useForm } from "react-hook-form";
+import Select from "react-select";
+import { GET_USER_FESTS } from "../graphql";
+import { GET_USERS } from "../graphql/get-users";
+import { CREATE_FEST } from "../graphql/mutations/create-fest/create-fest";
+import { useModals } from "../store";
+import { CurrentUserContext } from "../store/current-user-context";
+import { Form, Modal } from "./index";
+
+export function SlopFestModal() {
+  const { data, loading, error } = useQuery(GET_USERS);
+  const [createFest, { loading: createLoading, error: createError }] =
+    useMutation(CREATE_FEST, { refetchQueries: [GET_USER_FESTS] });
+  const {
+    register,
+    handleSubmit,
+    formState: { isValid, errors },
+    setValue,
+    getValues,
+  } = useForm({
+    defaultValues: {
+      name: "",
+      startDate: "",
+      endDate: "",
+      attendees: [],
+    },
+  });
+  const [startDate, setStartDate] = useState(new Date());
+  const [endDate, setEndDate] = useState(new Date());
+  const [users, setUsers] = useState([]);
+  const { currentUser } = useContext(CurrentUserContext);
+  const { closeModal } = useModals();
+
+  useEffect(() => {
+    if (data && data.users) {
+      setUsers(data.users);
+    }
+  });
```

Comment:
Consider using a dependency array here for for data to keep users up to date with the Apollo Client:
useEffect(() => {
  if (data && data.users) {
    setUsers(data.users);
  }
}, [data]); // Add data as a dependency